### ISSUE TITLE
How to use the early terminate module in wandb? I do not quite understand the explaination in docs.

### ISSUE THREAD

**Issue by zlq147** [wandb/wandb#4481]
I am trying to use wandb sweep to tune the hyperparameter in a model, and also try to use the hyperband early terminate method to accelerate it. 

However, I don't understand how this mechanism works by looking up the docs https://docs.wandb.ai/guides/sweeps/define-sweep-configuration#early_terminate and the paper https://arxiv.org/abs/1603.06560. 

In this paper, the author propose the concept of "resource". In my opinion, in the wandb setting, the "resource" should be num of training epochs. However, in the configuration of "early terminate", I can only see the parameter of "s", "eta", "min_iter" and "max_iter". And through the explaination of the docs, I do not understand the real meaning of them. 

In the github examples, it is tough to see whether the early terminate takes effect, so I hope there will be a simple piece of code to explain how the early terminate works. I wonder if the logged metric shourld be "valid_acc". 

I would be appreciated if anyone can help me understand what early terminate mechanism in wandb sweep actually do, especially the meaning of the parameters, and how to change the training code.

**Comment by luisbergua** [comment]
Hi @zlq147, thanks for your question! [Here](https://open.gitcode.host/wandb-docs/sweeps/configuration.html#stopping-criteria) you have a more detailed explanation of the early termination, please let me know if this would be useful or if you have any other question.

**Comment by luisbergua** [comment]
Hi @zlq147, I wanted to follow up here! Was [this](https://open.gitcode.host/wandb-docs/sweeps/configuration.html#stopping-criteria) information useful? Do you have any other question about this?

**Comment by luisbergua** [comment]
Hi @rbracco, sorry for the long delay here! Just wanted to let you know that I submitted your feedback internally to improve our docs and better explain how the early terminate module works. Thanks a lot for sharing the detailed explanation!

**Comment by JackCai1206** [comment]
I'd also love to see some concrete examples without having to dive into the paper!

**Comment by royvelich** [comment]
Same here. The docs are not clear IMHO.

**Comment by ziimiin14** [comment]
Hi @luisbergua , just a follow up. Is there any progress on this issue? Facing same issue as stated above.

**Comment by SuroshAhmadZobair** [comment]
Hi

@luisbergua  any updates?

**Comment by luisbergua** [comment]
Hi @SuroshAhmadZobair @ziimiin14, apologies for the delay. I'll bump the priority of this with our Docs Team
